stable_baselines/common/policies.py

```diff
@@ -25,6 +27,65 @@ def nature_cnn(scaled_images, **kwargs):
     return activ(linear(layer_3, 'fc1', n_hidden=512, init_scale=np.sqrt(2)))
 
 
+def mlp_extractor(flat_observations, net_arch, act_fun):
+    """
+    Constructs an MLP that receives observations as an input and outputs a latent representation for the policy and
+    a value network. The `layers_spec` parameter allows to specify the amount and size of the hidden layers and how many
+    of them are shared between the policy network and the value network. It is assumed to be a list with the following
+    structure:
+
+    1. An arbitrary length (zero allowed) number of integers each specifying the number of units in a shared layer.
+       If the number of ints is zero, there will be no shared layers.
+    2. An optional dict, to specify the following non-shared layers for the value network and the policy network.
+       It is formatted like `dict(vf=[<value layer sizes>], pi=[<policy layer sizes>])`.
+       If it is missing any of the keys (pi or vf), no non-shared layers (empty list) is assumed.
+
+    For example to construct a network with one shared layer of size 55 followed by two non-shared layers for the value
+    network of size 255 and a single non-shared layer of size 128 for the policy network, the following layers_spec
+    would be used: `[55, dict(vf=[255, 255], pi=[128])]`. A simple shared network topology with two layers of size 128
+    would be specified as [128, 128].
+
+    :param flat_observations: The observations to base policy and value function on.
```

Comment:
```suggestion
    :param flat_observations: (tf.Tensor) The observations to base policy and value function on.
```
Please specify the type for the others inputs